packages/ringcentral-widgets/components/InsideModal/index.js

```diff
@@ -5,37 +5,17 @@ import Modal from '../Modal';
 import styles from './styles.scss';
 import InsideModalClose from '../../assets/images/InsideModalClose.svg';
```

Comment:
Seems like can remove this line at the same time.

Reply:
Done!